tests/integration/cli_test.py

```diff
@@ -5,9 +5,9 @@
 import sys
 from operator import attrgetter
 
-from mock import patch
 from six import StringIO
 
+from .. import mock
```

Comment:
What's the context behind not importing `patch`? I'd rather `patch` be specifically imported then the whole of `mock`, it's more explicit what we're using then.